src/middleware/devtools.ts

```diff
@@ -203,6 +213,17 @@ const devtoolsImpl: DevtoolsImpl =
     const { connection, ...connectionInformation } =
       extractConnectionInformation(store, extensionConnector, options)
 
+    const isActionDenylisted = (action: { type: string }): boolean => {
+      if (!actionsDenylist) return false
+      if (Array.isArray(actionsDenylist)) {
+        return actionsDenylist.includes(action.type)
+      }
+      if (typeof actionsDenylist === 'function') {
+        return actionsDenylist(action)
+      }
+      return false
+    }
```

Comment:
Do you want to support plain string list and function filter as opposed to be compatible with Redux one (regex string and array of regex string)?